### ISSUE TITLE
Support for python3

### ISSUE THREAD

**Issue by shivammutreja** [getsentry/sentry-phabricator#12]
Most of the python projects are now being written using python3.
Support for Python 3 will be greatly appreciated.

**Comment by markstory** [comment]
Any chance you'd be interested in helping getting this or other plugins compatible with Python3?

**Comment by shivammutreja** [comment]
Yes, I'll be happy to help in getting this compatible with Python3 for starters.
